### ISSUE TITLE
📎 CSS Grid support

### ISSUE THREAD

**Issue by ematipico** [rome/tools#1556]
Supporting CSS grid requires a bit a work involved. There are different functions and properties. Also, it has a restricted grammar. 

Here's what I found so far:

- [x] [`repeat()`](https://csstree.github.io/docs/syntax/#Function:repeat)
- [x] [`minmax()`](https://csstree.github.io/docs/syntax/#Type:minmax())
- [x] [`fit-content()`](https://csstree.github.io/docs/syntax/#Function:fit-content)
- [ ] [`grid`](https://csstree.github.io/docs/syntax/#Property:grid)

Refers to #984

**Comment by ktfth** [comment]
That task still available to work on? What is the steps to make a better contribution, what is need to see to made the work?

**Comment by ematipico** [comment]
@ktfth you could try with `fit-content`. The spec is quite simple.

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
